mne/channels/montage.py

```diff
@@ -862,6 +903,7 @@ def _backcompat_value(pos, ref_pos):
 
         for name, use in zip(info_names, info_names_use):
             _loc_view = info['chs'][info['ch_names'].index(name)]['loc']
+            # XXX info['chs'][#]['loc'] modified in place
```

Comment:
It is not directly related but I wrote those comments to pinpoint all the lines where the in-place modifications were happening as it wasn't always obvious at first glance for example for this line 907.

Reply:
Should I remove them or is it okay?

Reply:
such comments are useful, but it would be clearer if it didn't start with `XXX` as this is usually a signal for "something to be fixed later / eventually".  Can you change them to say "next line modifies info['chs'][#]['loc'] in-place" or similar?